ros_packages/motors/launch/launch.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+from launch_ros.actions import Node
+from pib_motors.update_bricklet_uids import *
+
+from launch import LaunchDescription
+
+
+def check_and_update_uids():
```

Comment:
function-name should be prefixed with an underscore

Reply:
added